modules/node_modules/@ncigdc/components/SSM.js

```diff
@@ -87,11 +88,9 @@ const Mutation = compose(
     c.node.transcript.annotation.dbsnp_rs ? c.node.transcript.annotation.dbsnp_rs : acc, '');
   const cosmic = consequences.reduce((acc, c) =>
     c.node.transcript.annotation.cosmic_id ? c.node.transcript.annotation.cosmic_id : acc, []);
-  const functionalImpact = (
-    consequences.reduce(
-      (impact, consequence) => impact || ((consequence.node.transcript || {}).annotation || {}).impact, ''
-    ) || ''
-  ).toLowerCase();
+  const consequenceOfInterest = consequences.find(consequence => get(consequence, 'node.transcript.annotation'), {});
```

Comment:
I think `consequences.find(c => _.get(c, 'node.transcript.annotation.impact'), {});` otherwise a consequence with `annotation: { impact: null }}` would be found before one with an impact. ie http://localhost:8080/ssms/1e745f6b-3e06-5fc8-aab0-afe3246fe5a9